script/eslint-plugin-va/rules/prefer-web-component-library.js

```diff
@@ -31,9 +31,10 @@ const telephoneTransformer = (context, node) => {
           return [
             fixer.replaceText(componentName, 'va-telephone'),
             stripHyphens &&
-              fixer.replaceText(
-                contactValue,
-                `'${contactValue.value.replace(/[^\d]/g, '')}'`,
+              fixer.replaceTextRange(
+                // Leave the quotes alone
+                [contactValue.range[0] + 1, contactValue.range[1] - 1],
+                contactValue.value.replace(/[^\d]/g, ''),
```

Comment:
This just replaces what's _inside_ the quotes rather than the entire quoted string. That way we don't have to worry about which quotes to use since certain rules can be picky about that.